example_test.go

```diff
@@ -154,7 +154,7 @@ func Example_put() {
 			Tags:    []string{"article", "sample", "resty"},
 		}).
 		SetAuthToken("C6A79608-782F-4ED0-A11D-BD82FAD829CD").
-		SetError(&Error{}). // or SetError(Error{}).
+		SetError(&Error{}). // or SetError(error{}).
```

Comment:
@tttturtle-russ The comment syntax does not need to be modified. If there is any reason, please don't hesitate to let me know.